assets/js/blocks/attribute-filter/block.js

```diff
@@ -215,19 +210,29 @@ const AttributeFilterBlock = ( {
 		[ attributeTerms ]
 	);
 
-	const onSubmit = () => {
-		if ( isEditor ) {
-			return;
-		}
+	const onSubmit = useCallback(
```

Comment:
Since `onSubmit` is a dependency of some effects, I wrapped it in a `useCallback()`.